test/server.ts

```diff
@@ -237,6 +247,98 @@ test('store server started in the background should use store location wanted by
   t.notOk(await pathExists(serverJsonPath), 'server.json removed')
 })
 
+async function testParallelServerStart (options: {
+  test: tape.Test,
+  timeoutMillis?: number,
+  onProcessClosed?: (serverProcess: ChildProcess, weAttemptedKill: boolean) => void,
+  n?: number,
+  }) {
+  const pnpmBin = 'pnpm'
+  const serverProcessList: ServerProcess[] = []
+  // Variable that allows us to cancel the timeout.
+  let timeoutPromise: PDelayedPassThroughThunk<void>|null = delay(!options.timeoutMillis && options.timeoutMillis !== 0 ? 10000 : options.timeoutMillis)
+  // Promise that completes when all server processes have terminated.
+  let completedPromise: Promise<void> = Promise.resolve()
+  const n = !options.n && options.n !== 0 ? 10 : options.n
+  for (let i = 0; i < n; i++) {
+    const item: ServerProcess = [
+      crossSpawn.spawn(
```

Comment:
You can use the spawn function from the test utils. If you use that function and you call the prepare function in the beginning of the test case, then you can skip setting the store location too

```js
spawn(['server', 'start'])
```